UnitTests/CCFileUtilTests.m

```diff
@@ -1,239 +1,239 @@
-            @"foo.wav" : @"foo.mp4"
-        },
-        @"metadata": @{
-            @"version" : @1
-        }
-    };
-    [lookup1 writeToFile:[pathToPackage stringByAppendingPathComponent:@"fileLookup.plist"] atomically:YES];
-
-    NSDictionary *lookup2 = @{
-        @"filenames": @{
-            @"baa.psd" : @"baa.png"
-        },
-        @"metadata": @{
-            @"version" : @1
-        }
-    };
-    [lookup2 writeToFile:[pathToPackage2 stringByAppendingPathComponent:@"fileLookup.plist"] atomically:YES];
-
-    [[CCFileUtils sharedFileUtils] loadFileNameLookupsInAllSearchPathsWithName:@"fileLookup.plist"];
-    NSDictionary *filenameLookup = [CCFileUtils sharedFileUtils].filenameLookup;
-    XCTAssertEqualObjects(filenameLookup[@"baa.psd"], @"baa.png");
-    XCTAssertEqualObjects(filenameLookup[@"foo.wav"], @"foo.mp4");
-}
-
--(void)testCCFileUtilsSearchModeSuffix
-{
-	CCFileUtils *sharedFileUtils = [CCFileUtils sharedFileUtils];
-	sharedFileUtils.searchMode = CCFileUtilsSearchModeSuffix;
-	
-	XCTAssertTrue([sharedFileUtils fullPathForFilename:@"Images/powered.png"] != nil, @"");
-}
-
--(void)testCCFileUtilsSearchModeDirectory
-{
-	CCFileUtils *sharedFileUtils = [CCFileUtils sharedFileUtils];
-	sharedFileUtils.searchMode = CCFileUtilsSearchModeDirectory;
-	
-	NSString *path = [sharedFileUtils fullPathForFilename:@"Images/powered.png" contentScale:nil];
-	XCTAssertTrue(path != nil, @"");
-	
-	CGFloat scale = 1.0;
-	CGFloat biggerScale = 2.0;
-	
-	path = [sharedFileUtils fullPathForFilename:@"Images/powered.png" contentScale:&scale];
-	XCTAssertTrue(path != nil, @"");
-	
-	path = [sharedFileUtils fullPathForFilename:@"Images/powered.png" contentScale:&biggerScale];
-	XCTAssertTrue(path != nil, @"");
-}
-
--(void)testCustomSearchPathsForExtensions
-{
-	CCFileUtils *sharedFileUtils = [CCFileUtils sharedFileUtils];
-	[sharedFileUtils purgeCachedEntries];
-	[sharedFileUtils setSearchPath: @[ @"Images", kCCFileUtilsDefaultSearchPath] ];
-	
-	XCTAssertTrue( [sharedFileUtils fullPathForFilename:@"Images/powered.png" contentScale:nil] != nil, @"");
-	XCTAssertTrue( [sharedFileUtils fullPathForFilename:@"powered.png" contentScale:nil] != nil, @"Search path 'Images' didn't work.");
-}
-
--(void)testContentScaleLoading
-{
-	CCFileUtils *sharedFileUtils = [CCFileUtils sharedFileUtils];
-	sharedFileUtils.searchMode = CCFileUtilsSearchModeSuffix;
-	
-	// assume we're on iPhone5 HD.
-	// CCFileUtilsSuffixiPhoneHD is hardcoded to look for the "-hd" suffix.
-	// So overwrite to look only for these for the purpose of the test.
-	[ (NSMutableArray*) [sharedFileUtils searchResolutionsOrder] removeAllObjects];
-	[ (NSMutableArray*) [sharedFileUtils searchResolutionsOrder] addObject:CCFileUtilsSuffixiPhoneHD];
-
-	CGFloat scale = 0.0;
-	
-	NSString *path1 = [sharedFileUtils fullPathForFilename:@"Images/blocks.png" contentScale:&scale];
-	XCTAssertTrue(scale == 2.0, @"");
-	XCTAssertTrue(path1 != nil, @"");
-}
-
-@end
+//#import <XCTest/XCTest.h>
```

Comment:
I would either remove this file or disable this test case within the project file, than commit commented out code to the repo